### DISCUSSION TITLE
Confusion about HammerDB source location and TPC association

### DISCUSSION THREAD

**Discussion by mikini** [TPC-Council/HammerDB#56] [Q&A] [answered]
I'm wondering about the subtlety with which [the TPC](http://tpc.org) has [announced on twitter](https://twitter.com/tpcbenchmarks/status/1131280447168753666) that HammerDB source code is now hosted in its Github organisation. The hammerdb.com homepage [still refers to sourceforge as the primary source location](https://www.hammerdb.com/about.html) (section "Where is the source code for HammerDB?").

Also the homepage advices not to [download from anywhere else than sourceforge](https://www.hammerdb.com/about.html) (section "Where do I download HammerDB?").

Adding to the confusion, the homepage has [previously pointed to](https://www.hammerdb.com/blog/2018/11/) a [sm-shaw github repository](https://github.com/sm-shaw/HammerDB) as being a mirror of the sourceforge git repository.

Looking at the latest activity in the three announced repositories it seems, however, that current work is actually happening in the sm-shaw repository without being mirrored to sourceforge nor TPC repository ([latest commit merging a pull request on January 21](https://github.com/sm-shaw/HammerDB/commit/7cc236e477365b508b285a9b1939b0c0d3df1613)).

Also I am puzzled by the lack of information about the association between HammerDB and the TPC. Apart from the Twitter announcment about the Github repository I haven't been able to dig up information about any association. The [documentation mentioning TPC](https://www.hammerdb.com/docs/ch03s02.html) doesn't disclose any association other than HammerDB implementing TPC-C from TPC. Looking into the [Steve Shaw Github user](https://github.com/sm-shaw) there is an apparent association with Intel according to a [Steve Shaw LinkedIn profile](https://www.linkedin.com/in/steve-shaw-80b6aa20).

**Comment by sm-shaw** [comment, marked as answer]
There is a [post on the HammerDB website](https://www.hammerdb.com/blog/uncategorized/hammerdb-source-code-moved-to-the-tpc-council/) to provide further information on the questions above. However the short answer is that the previous sm-shaw github repository has been deleted and this TPC Council repository is the development site for ongoing and future HammerDB development.  As development has previously been on sourceforge there will be a period of transition of all functionality such as binary downloads between the two to coincide with new releases.
